docs/contributing.rst

```diff
@@ -75,9 +75,16 @@ The NumCodecs source code is hosted on GitHub at the following location:
 You will need your own fork to work on the code. Go to the link above and hit
 the "Fork" button. Then clone your fork to your local machine::
 
-    $ git clone git@github.com:your-user-name/numcodecs.git
+    $ git clone --recursive git@github.com:your-user-name/numcodecs.git  # with ``ssh``
+    
+or::
+
+    $ git clone --recursive https://github.com/your-user-name/numcodecs.git  # with ``https``
+
+Then ``cd`` into the clone and add the ``upstream`` remote::
+
     $ cd numcodecs
-    $ git remote add upstream git@github.com:zarr-developers/numcodecs.git
+    $ git remote add upstream git@github.com:zarr-developers/numcodecs.git  # or use ``https``
```

Comment:
Actually would just use `https` here. It prevent pushing upstream when using `ssh`, which (to me at least) is a feature and not a bug :)

```suggestion
    $ git remote add upstream https://github.com/zarr-developers/numcodecs.git
```